### ISSUE TITLE
Scons doesn't find the (installed) Intel fortran compiler

### ISSUE THREAD

**Issue by bdbaddog** [SCons/scons#2752]
This issue was originally created at: 2011-04-16 10:36:31.
This issue was reported by: `charlesli`.
charlesli said at 2011-04-16 10:36:31
>Although I have the Intel Fortran compiler installed (version 11.1, evaluation version), scons doesn't find it. Instead it tries to use gfortran (and fails).

**Comment by dnwillia** [comment]
I can certainly confirm this is still an issue on both Windows and Linux.  The examples in my repo here:

https://github.com/dnwillia/SConsTests

Will not compile out of the box if the `ifort` tool is added to the environment.

**Comment by mwichmann** [comment]
not surprised... the Intel compilers have moved and morphed over the years.

**Comment by dnwillia** [comment]
Yeah, keeping track of their constantly changing directory structure is a little painful.

One possibility might just be to make it work with `PrependENVPath`.  So, if I put the `ifort` or `ifx` binary location into the environment path the tool can detect it and work out the include and linker paths from that.  Not sure though.

There are also Intel's configuration scripts.  This might be another way to do it.  i.e. run their setup script to configure the environment.
